Example_configs.md

```diff
@@ -0,0 +1,289 @@
+# Example Configs
+
+## ubuntu
+Example configuration file to build both the Kernel module and eBPF probe for Ubuntu (works with any flavor!).
+
+```yaml
+kernelrelease: 5.0.0-1021-aws-5.0
+kernelversion: 24~18.04.1
+target: ubuntu
+output:
+  module: /tmp/falco-ubuntu-generic.ko
+  probe: /tmp/falco-ubuntu-generic.o
+driverversion: master
+```
+
+## ubuntu-generic
+Example configuration file to build both the Kernel module and eBPF probe for Ubuntu generic.
+
+```yaml
+kernelrelease: 4.15.0-72-generic
+kernelversion: 81
+target: ubuntu-generic
```

Comment:
```suggestion
target: ubuntu
```
As the target `ubuntu-generic` is the previous format, maybe we should doc only the new one ?

Reply:
We should, at least, indicate that this is the old format.

Reply:
Agree, perhaps a note can be helpful there. We can remove these altogether too IMO!
@dwindsor what do you think?

Reply:
I think we should probably start deprecating the old `target` formatting before our 1.0 release, so a note would be good! @FedeDP do you mean remove this file altogether?

Reply:
Uh no, i meant remove the lines about `ubuntu-aws` and `ubuntu-generic` :)

Reply:
Added the notes!